spec/user_can_delete_game_spec.rb

```diff
@@ -0,0 +1,32 @@
+require 'rails_helper'
+
+feature 'only admin or users can delete a game' do
+  let(:game) { FactoryGirl.create(:game) }
+  scenario 'user can delete game' do
+    sign_up
+    visit new_game_path
+    fill_in 'name', with: 'wowgame'
+    fill_in 'description', with: 'this game is great'
+    fill_in 'minimum players', with: 2
+    fill_in 'maximum players', with: 8
+    fill_in 'playing time (min)', with: 50
+    select "medium", :from => "complexity"
+    click_button "add game"
+    click_link "edit game"
+    click_button "delete this game"
+    expect(page).to have_content("Game deleted.")
+  end
+end
+
+def sign_up
+  visit new_user_registration_path
+  fill_in 'email', with: 'john@example.com'
+  fill_in 'password', with: 'password'
+  fill_in 'confirmation', with: 'password'
+  fill_in 'username', with: 'im a user'
+
+  click_button 'Sign up'
+
+  expect(page).to have_content('Welcome! You have signed up successfully.')
+  expect(page).to have_content('Sign Out')
```

Comment:
You wouldn't have your method expect a page to have content. Your tests should test for pages having content, not specific methods.